### PR TITLE
Add config parameter to disable bridging m.notice-type messages

### PR DESCRIPTION
**Pull request by jzapataikono**
Added feature for m.notice-type messages not be bridged

### CODE REVIEW

File: mautrix_facebook/example-config.yaml [hunk 1]
```diff
@@ -257,6 +257,8 @@ bridge:
     temporary_disconnect_notices: false
     # Disable bridge notices entirely
     disable_bridge_notices: false
+    # Should Matrix m.notice-type messages be bridged?
+    bridge_notices: true
```
Comment (sumnerevans) [inline mautrix_facebook/example-config.yaml:261]: maybe this makes it clearer which direction is being talked about?

```suggestion
    # Should Matrix m.notice-type messages be bridged to Facebook?
    bridge_matrix_notices: true
```

Comment (jzapataikono) [reply mautrix_facebook/example-config.yaml]: You are right, I applied the suggested changes

**Comment by jzapataikono** [comment]
Hi, what do I need to have this PR accepted ?

File: mautrix_facebook/portal.py [hunk 2]
```diff
@@ -1276,6 +1276,13 @@ async def _handle_matrix_text(
     ) -> None:
         converted = await matrix_to_facebook(message, self.mxid, self.log)
         dbm = await self._make_dbm(sender, event_id)
```
Comment (tulir) [inline mautrix_facebook/portal.py:1278]: These should probably happen after the check

**Review by tulir — APPROVED** [review]

**Pull request merged**